### DISCUSSION TITLE
Back up from my self-hosted minio s3

### DISCUSSION THREAD

**Discussion by Sina-Najafi** [minio/minio#18483] [Q&A] [answered]
Hello,
I have a self-hosted minio s3 which I am using to store some data in different buckets. I also have multiple users and policies too. I would like to back up all my data and user and policies and other configurations that I have in my self hosted minio and put it into my Amazon s3 glacier. What is the right way of doing it? 
I would be really grateful if you could help me...

**Comment by harshavardhana** [comment, marked as answer]
https://min.io/docs/minio/kubernetes/upstream/administration/concepts.html#how-can-i-backup-and-restore-objects-on-minio

**Reply by voidus** [reply]
link is dead
